template.hbs

```diff
@@ -229,8 +229,7 @@
       {{#each Certificate.CommercialTransaction.SupplementaryInformation}}
       <tr class="table-rows">
         <th class="table-headers" colspan="2">{{@key}} {{this.Key}}</th>
-        <td class="table-cells">{{ localizeValue this.Value this.Type ../Certificate.CertificateLanguages.[0] }}
-          {{this.Unit}}</td>
+        <td class="table-cells">{{localizeValue this.Value ../../Certificate.CertificateLanguages.[0]}} {{this.Unit}}</td>
```

Comment:
For all places where `localizeValue` helper is used we need `this.Type` to be passed too